ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -655,6 +679,7 @@ def traced_chain_stream(langchain, pin, func, instance, args, kwargs):
     integration: LangChainIntegration = langchain._datadog_integration
 
     def _on_span_started(span: Span):
+        integration.record_instance(instance, span)
```

Comment:
Does this need to be `record_steps`?

Reply:
yes good catch!

Reply:
I actually ended up removing `record_steps` - tl;dr we might not always want to record a step as a "chain step" directly, as something like a model could be re-used standalone. I replaced checking against `self._chain_steps` with checking if the parent instance has a `steps` attribute